.github/workflows/benchmark.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     #- name: Run Stable Diffusion
     #  run: python3 examples/stable_diffusion.py --seed 0 --noshow --timing | tee sd.txt
     - name: Run model inference benchmark
-      run: METAL=1 python3 test/external/external_model_benchmark.py
+      run: METAL=1 NOCLANG=1 python3 test/external/external_model_benchmark.py
```

Comment:
Before the benchmark tests didn't actually perform device specific validation tests. 
`[Device.DEFAULT, "CLANG"]` made it so the last `Device.DEFAULT` was set to `CLANG` and validation was done to `CLANG` vs `CPUExecutionProvider`

Reply:
cool, can we have a separate pr that just fixes this?